assets/large_language_models/prompts/generate_a_discharge_letter/prompt/prompt.yaml

```diff
@@ -0,0 +1,122 @@
+    Treatment Provided:
+    - IV Ceftriaxone 2g daily
+    - IV Azithromycin 500mg daily
+    - Metformin 1000mg twice daily for diabetes
+    Condition at Discharge: Improved. Patient reports no more fever or difficulty breathing. Cough significantly reduced.
+    Discharge Medications:
+    - Amoxicillin 500mg three times daily for 7 days - Antibiotic for pneumonia
+    - Metformin 1000mg twice daily - For diabetes control
+    Follow-up care:
+    - Outpatient clinic follow-up in 2 weeks for symptom check and blood tests
+    - Home health nursing for medication management and diabetes education
+    - Continue monitoring blood glucose levels regularly
+    - Maintain a balanced, low-sugar diet
+    - Regular exercise as tolerated
+- role: assistant
+  content: |
+    Discharge Letter:
+    Mary Jones, born April 14, 1985, was admitted to the hospital on July 1, 2023, due to pneumonia, with a secondary diagnosis of Type 2 Diabetes Mellitus. Her chest X-Ray showed a right lower lobe consolidation and sputum culture identified Streptococcus pneumoniae. She was treated with IV ceftriaxone, azithromycin, and metformin for diabetes during her stay. Upon discharge on July 8, 2023, her condition had improved significantly with reduced cough and no fever. She was prescribed amoxicillin for the continued treatment of pneumonia and metformin for diabetes management. She was advised to maintain a balanced, low-sugar diet and regular exercise, with a follow-up appointment scheduled in two weeks. Home health nursing was arranged for medication management and diabetes education.
+    Regards, John Wick
+- role: user
+  content: |
+    Write a discharge letter for the patient with following information:
+    Patient's Name: {{patient_name}}
+    Patient's Date of Birth: {{dob}}
+    Date of Admission: {{date_of_admission}}
+    Date of Discharge: {{date_of_discharge}}
+    Medical Record Number: {{medical_record_number}}
+    Doctor's Name: {{doctor_name}}
+    Primary Diagnosis: {{primary_diagnosis}}
+    Secondary Diagnosis: {{secondary_diagnosis}}
+    Reason for Hospitalization: {{reason_for_hospitalization}}
+    Significant Findings: {{significant_findings}}
+    Procedures Performed: {{procedures_performed}}
+    Treatment Provided: {{treatment_provided}}
+    Condition at Discharge: {{condition_at_discharge}}
+    Discharge Medications: {{discharge_medications}}
+    Follow-up care: {{follow_up_care}}
+
+input_variables:
+- name: condition_at_discharge
+  description: Condition at Discharge
+  defaultValue: |
+    Stable. Patient experienced significant symptom relief post-PCI. No further episodes of chest pain or shortness of breath. Blood pressure under control with medications.
+- name: dob
+  description: DOB
+  defaultValue: January 1, 1960
+- name: date_of_admission
+  description: Date of Admission
+  defaultValue: June 1, 2023
+- name: date_of_discharge
+  description: Date of Discharge
+  defaultValue: June 7, 2023
```

Comment:
let's rename to simply default. Sorry it was a miss from my side

Reply:
fixed